src/DynamicAttributes/DynamicAttributes.php

```diff
@@ -0,0 +1,47 @@
+<?php
+declare(strict_types=1);
+
+namespace Thinktomorrow\Chief\DynamicAttributes;
+
+class DynamicAttributes
+{
+    /** @var array */
+    private $values;
+
+    public function __construct(array $values)
+    {
+        $this->values = $values;
+    }
+
+    public static function fromRawValue($value): self
+    {
+        $value = is_null($value) ? [] : (is_array($value) ? $value : json_decode($value, true));
+
+        return new static((array) $value);
+    }
+
+    public function has(string $key): bool
+    {
+        return ($this->get($key, '__NOTFOUND__') !== '__NOTFOUND__');
```

Comment:
Is dit op deze manier gedaan omdat een value met waarde null wel aanzien moet worden als 'existing'?

Reply:
ja inderdaad. :)